integration/container/restart_test.go

```diff
@@ -76,11 +76,10 @@ func TestDaemonRestartKillContainers(t *testing.T) {
 					d.Stop(t)
 				},
 			} {
+				c := tc
```

Comment:
- We commonly use `tc := tc` for these. Could be worth considering here as well.
- Wondering if it's more accurate for this this to be in the "outer" loop 🤔

Reply:
- Changed `c` to `tc`.
- It would be more accurate in theory, but in practice IMO it's more visible when it's just before `Run`.